presto-main/src/test/java/com/facebook/presto/metadata/TestDiscoveryNodeManager.java

```diff
@@ -71,27 +75,68 @@ public void setup()
         testHttpClient = new TestingHttpClient(input -> new TestingResponse(OK, ArrayListMultimap.create(), ACTIVE.name().getBytes()));
 
         expectedVersion = new NodeVersion("1");
-        coordinator = new InternalNode(UUID.randomUUID().toString(), URI.create("https://192.0.2.8"), expectedVersion, true);
-        resourceManager = new InternalNode(UUID.randomUUID().toString(), URI.create("https://192.0.2.9"), expectedVersion, false, true);
-        currentNode = new InternalNode(nodeInfo.getNodeId(), URI.create("http://192.0.1.1"), expectedVersion, false);
-
-        activeNodes = ImmutableSet.of(
-                currentNode,
+        coordinator = new InternalNode(coordinatorNodeInfo.getNodeId(), URI.create("https://192.0.2.8"), expectedVersion, true);
+        inActiveCoordinator = new InternalNode(UUID.randomUUID().toString(), URI.create("https://192.0.3.1"), new NodeVersion("2"), true);
+        resourceManager = new InternalNode(resourceManagerNodeInfo.getNodeId(), URI.create("https://192.0.2.9"), expectedVersion, false, true);
+        inActiveResourceManager = new InternalNode(resourceManagerNodeInfo.getNodeId(), URI.create("https://192.0.2.9"), new NodeVersion("2"), false, true);
+        workerNode = new InternalNode(workerNodeInfo.getNodeId(), URI.create("http://192.0.1.1"), expectedVersion, false);
+
+        workerNodes = ImmutableSet.of(
+                workerNode,
                 new InternalNode(UUID.randomUUID().toString(), URI.create("http://192.0.2.1:8080"), expectedVersion, false),
-                new InternalNode(UUID.randomUUID().toString(), URI.create("http://192.0.2.3"), expectedVersion, false),
-                coordinator,
-                resourceManager);
+                new InternalNode(UUID.randomUUID().toString(), URI.create("http://192.0.2.3"), expectedVersion, false));
+        activeNodes = ImmutableSet.<InternalNode>builder()
+                .addAll(workerNodes)
+                .add(coordinator)
+                .add(resourceManager)
+                .build();
         inactiveNodes = ImmutableSet.of(
+                inActiveCoordinator,
+                inActiveResourceManager,
                 new InternalNode(UUID.randomUUID().toString(), URI.create("https://192.0.3.9"), NodeVersion.UNKNOWN, false),
                 new InternalNode(UUID.randomUUID().toString(), URI.create("https://192.0.4.9"), new NodeVersion("2"), false));
```

Comment:
make them as inActiveWorkerNode1, inActiveworkerNode2 or something of that kind to make it consistent with other variable.